openage/convert/processor/export/generate_hash.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020-2020 the openage authors. See copying.md for legal info.
```

Comment:
Small suggestion: Rename this file to `generate_manifest_hashes.py`, so it's more clear what we are doing here :)